steps/parameterList.ts

```diff
@@ -4,4 +4,12 @@ export const parameterDescriptions = {
   withoutLanguageVariants: 'Exclude languages with Country variant. For example `en_CA` will not be present in zim with this argument.',
   mulOnly: 'Skip ZIM files for individual languages',
   createMul: 'Create a ZIM file with all languages',
+  output: 'Output ZIM files in a specific directory',
```

Comment:
Something like `Directory where ZIM files will be written` would make it more obvious that user need to pass a directory ; other parameter are boolean so using an imperative form is necessary, here we need a value.